app/templates/content.hbs

```diff
@@ -79,7 +79,12 @@
                     </div>
                     <div class="p-t-xs">
                         <h4 class="p-v-md f-w-md"><strong>{{t "global.abstract"}}</strong></h4>
-                        <p class="abstract">{{node.description}}</p>
+                        {{#if useShortenedDescription}}
+                            <p class="abstract">{{description}}</p>
+                            <button class='btn-link' {{action 'expandAbstract'}}>{{if expandedAbstract 'See less' 'See more'}}</button>
```

Comment:
In the future, these strings need to go into the translations file, not in the template.